### PR TITLE
Email Validation

### PR DESCRIPTION
**Pull request by Navaneeth-84**
simple login form

### CODE REVIEW

File: login-form/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          <div class="msgArea uname"></div>
+        </div>
+        <div class="element">
+          password
+          <div><input type="password" class="password" /></div>
+
+          <div class="msgArea pwd"></div>
+        </div>
+        <div class="element">
+          <button class="login-btn" onclick="validateFieldsNotEmpty()">
```
Comment (ksrscharan) [inline login-form/index.html]: onclick inside of html isn't something we should practice. Handling events in JS is a better way.

File: login-form/script.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    errorAlert("invalid password", ".pwd");
+  } else if (isValidPassword == true && isValidUsername == true) {
+    alert("valid username and pass");
+  }
+}
+
+
+
+
+// == vs ===
```
Comment (ksrscharan) [inline login-form/script.js]: understand difference between `==` and `===`.

File: login-form/style.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  font-size: 16px;
+  margin: 4px;
+  min-width: 240px;
+}
+
+.pwdRules {
+  margin: 20px;
+  height: 60px;
+  width: 200px;
+}
```
Comment (ksrscharan) [inline login-form/style.css:107]: Revamp UI design to resemble preview image.

File: login-form/index.html (hunk 1 above)
Comment (Navaneeth-84) [reply login-form/index.html]: image

image

been with this for a while, 
why this doesn't work ?? 😒
the function doesn't invoke when clicked.

Comment (Navaneeth-84) [reply login-form/index.html]: @ksrscharan

Comment (ksrscharan) [reply login-form/index.html]: Implemented the same technique. just check code @Navaneeth-84 .

Comment (Navaneeth-84) [reply login-form/index.html]: so silly of me to place script into the `head` tag. got it 😁

File: login-form/style.css (hunk 3 above)
Comment (Navaneeth-84) [reply login-form/style.css:107]: done

File: login-form/index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            <div><input type="text" class="username" /></div>
+            <div class="msgArea email"></div>
+          </div>
+          <div class="passwd">
+            <div><label>Password</label></div>
+            <div><input type="password" class="password" /></div>
+            <div class="msgArea pwd"></div>
+          </div>
+          <div class="forgot desc">
+            <div><a href="#">Forgotten Password?</a></div>
```
Comment (ksrscharan) [inline login-form/index.html:26]: why 2 `div` tags?

File: login-form/index.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+      <div class="loginform flexcol">
+        <div class="heading">Amplify</div>
+        <div class="fields desc">
+          <div class="uname">
+            <div><label>Username or Email</label></div>
+            <div><input type="text" class="username" /></div>
+            <div class="msgArea email"></div>
+          </div>
+          <div class="passwd">
+            <div><label>Password</label></div>
```
Comment (ksrscharan) [inline login-form/index.html:21]: why 2 `div` tags?

File: login-form/index.html [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container flexcol">
+      <div class="loginform flexcol">
+        <div class="heading">Amplify</div>
+        <div class="fields desc">
+          <div class="uname">
+            <div><label>Username or Email</label></div>
+            <div><input type="text" class="username" /></div>
```
Comment (ksrscharan) [inline login-form/index.html:17]: why 2 `div` tags?

File: login-form/index.html [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    <title>login page</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container flexcol">
+      <div class="loginform flexcol">
+        <div class="heading">Amplify</div>
+        <div class="fields desc">
+          <div class="uname">
+            <div><label>Username or Email</label></div>
```
Comment (ksrscharan) [inline login-form/index.html:16]: why 2 `div` tags?

File: login-form/index.html [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+            <div><input type="text" class="username" /></div>
+            <div class="msgArea email"></div>
+          </div>
+          <div class="passwd">
+            <div><label>Password</label></div>
+            <div><input type="password" class="password" /></div>
```
Comment (ksrscharan) [inline login-form/index.html:22]: why 2 `div` tags?

Comment (ksrscharan) [reply login-form/index.html:22]: Instead, try adding
```
.password{
display: inline-block;
}
```

**Comment by ksrscharan** [comment]
Everything else looks good. LGTM.

File: login-form/index.html (hunk 4 above)
Comment (Navaneeth-84) [reply login-form/index.html:26]: forgot password is into a diff flex area on a whole

File: login-form/index.html (hunk 5 above)
Comment (Navaneeth-84) [reply login-form/index.html:21]: passwd child to login form div and the label and its text box are children to passwd div, I'll see to it if I could reduce anything.

File: login-form/index.html (hunk 8 above)
Comment (Navaneeth-84) [reply login-form/index.html:22]: ok

**Review by ksrscharan — APPROVED** [review]

**Pull request merged**